benchmarks/benchmark_shared.c

```diff
@@ -192,3 +192,92 @@ int bandwidth(void)
 
 	return 0;
 }
+
+int bandwidth_rma(enum ft_rma_opcodes rma_op, struct fi_rma_iov *remote)
+{
+	int ret, i, j;
+
+	ret = ft_sync();
+	if (ret)
+		return ret;
+
+	if (!ctx_arr) {
+		return -FI_ENOMEM;
+	}
```

Comment:
Sure. I'm also moving ctx_arr allocation to ft_alloc_msgs so that they can be reused when the same test is run for different message sizes.